test/unit/alignment/pairwise/fixture/global_affine_unbanded.hpp

```diff
@@ -503,4 +503,138 @@ static auto aa27_blosum62_gap_1_open_10 = [] ()
     };
 }();
 
+static auto aa27_blosum62_gap_1_open_10_small = [] ()
```

Comment:
Just a question, but can't you use an existing alignment and trim it?

For example, I used:

```c++
static auto dna4_02T_s15u_1u = []()
{
    return alignment_fixture
    {
        // score: 14 (14 insertions)
        // alignment:
        // --------------
        //
        // AACCGGTAAACCGG
        ""_dna4,
        "AACCGGTAAACCGG"_dna4,
        seqan3::align_cfg::method_global{} | seqan3::align_cfg::edit_scheme,
        -14,
        "--------------",
        "AACCGGTAAACCGG",
        seqan3::alignment_coordinate{column_index_type{0u}, row_index_type{0u}},
        seqan3::alignment_coordinate{column_index_type{0u}, row_index_type{14u}},
        dna4_02.score_matrix().transpose_matrix().sub_matrix(15u, 1u),
        dna4_02.trace_matrix().transpose_matrix().sub_matrix(15u, 1u)
    };
}();
```

In my edit fixtures to simulate empty first sequence.

Reply:
Not sure what you mean. In general, when I change the sequence I need to adapt all the other parameters anyway.

Reply:
Yeah, but why don't we just use sub-sequences of existing known alignments?

This makes it easier to verify the correctness.

Reply:
I am still not following. If I use the subsequence if another test fixture than all I saved was the sequence. Everything else must be overriden. The score matrix the trace matrix and the coordinates. So what would I safe? And what would be better to verify for correctness then?